pages/clubs-guides/clubs-guides.jsx

```diff
@@ -0,0 +1,96 @@
+var React = require('react');
+var HeroUnit = require('../../components/hero-unit.jsx');
+// use this LinkAnchorSwap component for hyperlinks
+var HeroUnit = require('../../components/hero-unit.jsx');
+
+var GuideList = React.createClass({
+  getInitialState: function(){
+    return {
+      guides : []
+    }
+  },
+  setData : function(data){
+    this.setState({guides : data });
+  },
+  getData : function(){
+    // This can be replaced with an AJAX request to an external JSON file
+    fetch('https://mozilla.github.io/learning-networks/clubs/clubs-resources.json').then(function(r){
```

Comment:
Oh, dang - do we have a 'go-to' approach to doing this? I've used jQuery in the past for ajax request. Would you recommend that in this case, too?